apps/saleor-app-checkout/.env

```diff
@@ -1,3 +1,14 @@
 NEXT_PUBLIC_SALEOR_API_URL=$SALEOR_API_URL
 NEXT_PUBLIC_CHECKOUT_APP_URL=$CHECKOUT_APP_URL
-SALEOR_APP_TOKEN=
+
+# Default values for tests
+# They will work only when replaying requests with Polly.js
+# To record new requests, use real credentials in .env.local
+TEST_MOLLIE_KEY=mollie_test_key
```

Comment:
Are you sure this will work? I think I came across a problem where Mollie does some kind of key validation in their SDK and they said it was malformed.

Reply:
I've checked it and it's not a problem with Mollie SDK - they don't do any validation in the SDK itself. It's actually a problem with our request interception: https://github.com/saleor/react-storefront/issues/310. Right now Polly.js doesn't intercept requests properly and the tests might fail if the `TEST_MOLLIE_KEY` is not configured with a real Mollie API key. I made a draft PR to fix this: https://github.com/saleor/react-storefront/pull/325

The goal of this PR hoewever was to fix the production build which was caused by importing `testingVars` so I guess we can leave it for now 😅